### PR TITLE
Style unifications

### PR DESCRIPTION
**Pull request by l0rinc**
Very good documentation, congrats to the team :)

While reading it, I found a few small inconsistencies and mistakes, thought I'll practice what I learned and send a PR to correct them :)

Separated fix types into separate commits.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -276,7 +276,7 @@ Consider using a weak copyleft license that won’t restrict the software that u
 
 You can can use GPL code — but only internally. Be sure it's a version of the GPL that continues to allow for the ASP loophole. AGPL and versions of the GPL with additional restrictions won't work.
 
-#####Who is the license owner?
+#####Who is the license two-factor?
```
Comment (l0rinc) [inline README.md:282]: Was renamed in the TOC, but not here

File: README.md [hunk 2]
```diff
@@ -316,7 +316,8 @@ Management can work with Legal to determine Intellectual Property concerns.
 
 Many package managers include a feature to make the applied license machine readable. Use these! An example for [Maven](https://maven.apache.org/pom.html#Licenses):
 
- ```<licenses>
+```xml
```
Comment (l0rinc) [inline README.md:326]: the opening tag was omitted before, as it was interpreted as the language of the code

File: README.md [hunk 3]
```diff
@@ -3,70 +3,70 @@ How to Open Source at Zalando
 
 A guide to releasing an open-source project at [Zalando](https://www.zalando.com>), Europe's largest online fashion platform.
 
-Table of Contents 
+Table of Contents
```
Comment (l0rinc) [inline README.md:6]: separated whitespace unifications to simplify the review process

File: README.md [hunk 4]
```diff
@@ -83,12 +83,12 @@ Because It Can:
 ##Our Open Source First Principles
 
 - **Do “Open Source First”**: If your Zalando project can also be useful to non-Zalandos, release it as open source from the start.
-- **Take Ownership**: Your team is responsible for ensuring that it’s possible to open-source your project. Your delivery lead is available for guidance.
+- **Take Ownership**: Your team is responsible for ensuring that it's possible to open-source your project. Your delivery lead is available for guidance.
```
Comment (l0rinc) [inline README.md:87]: both types were used, i.e. `it's` and `it’s`

**Comment by hjacobs** [comment]
@paplorinc thanks for the PR, I think it was overlooked somehow --- will review & merge.. :hourglass_flowing_sand:

**Comment by l0rinc** [comment]
K, would you like me to resolve the conflicts?

**Comment by lasomethingsomething** [comment]
@paplorinc If it's possible, that would be great.

File: README.md (hunk 1 above)
Comment (hjacobs) [inline README.md:282]: but "owner" is correct

File: README.md [hunk 5]
```diff
@@ -125,98 +125,97 @@ Creating a new open source project should almost always be a team decision. This
...
-
-####The Incubator: For the Open-Source, But Not "Truly Open Source"
-Projects that aren't "truly open source":
-- are dependent on our systems
-- are dependent on other Zalando open-source projects that haven't been widely adopted
-- are “experimental”/”maybe useful to others someday,” but you're not working on it (a Hack Week project, or something you've lost interest in)
-- lacking meaningful documentation (a README), and you're not working on any
-
-These are currently not suitable for the main org repo. Place them at github.com/zalando-incubator. The Incubator reiterates the Open Source First principle, "Share Your Code: All code shared between teams must be open source." Even if Incubator projects aren't currently useful outside of Zalando, we'd like to think that one day, they might be.
+###The Main Zalando Repository: For the “Truly Open Source”
```
Comment (hjacobs) [inline README.md:144]: maybe use simple ASCII double quotes (") everywhere instead?

File: README.md (hunk 1 above)
Comment (l0rinc) [inline README.md:282]: It seemed to me that way also, please correct it in the TOC also in that case

File: README.md (hunk 5 above)
Comment (l0rinc) [inline README.md:144]: Sure, just make it the same everywhere ... though this separates real quotations in English from Strings in a given programming language

**Pull request closed without merging**